nss-capricious-finches/nutshell
# Created sign up form minus posting to API for Aloha

## Link to Ticket
Closes [#15](https://github.com/nss-capricious-finches/nutshell/issues/15)

## Description of Proposed Changes
1. Created a sign up form that follows Bulma class standards
1. Created newUserSignUp.js
1. Function named signUpForm() that appends html to class="signUp"
- 

## Steps to Test
1. Fetch and checkout correlating branch
1. Check newUserSIgnUp.js
1. Check to make sure button named "sign up" is console logging on click
1. Created images directory with brand logo inside
1. Added modularity by creating signUp.css to hold styling for JS file.

```
git fetch --all
git checkout JS_newUserSignUp
```
